pkgs/dartpad_ui/lib/enable_gen_ai.dart

```diff
@@ -5,6 +5,12 @@
 // Turn on or off gen-ai features in the client.
 const bool genAiEnabled = true;
 
+// Set to true to use GenUI instead of Gemini.
+// This is a temporary flag to test GenUI in production.
+// It will be removed once GenUI is fully integrated.
+// The flag is set based on URL parameter.
+bool useGenui = false;
```

Comment:
Always hardcoded to `false`? Is the comment about the URL parameter still true? Where is the value read now?

Reply:
Returned it to main. Thank you.